rules-tests/TypeDeclaration/Rector/ClassMethod/AddParamTypeBasedOnPHPUnitDataProviderRector/Fixture/fixture.php.inc

```diff
@@ -38,6 +43,11 @@ final class SomeTestWithDataProvider extends TestCase
     {
     }
 
+    #[\PHPUnit\Framework\Attributes\DataProvider('provideData')]
+    public function test_with_attribute(string $name)
+    {
+    }
```

Comment:
Could you separate these changes to a new fixture file? It will be easier to connect together in years ahead blame

Reply:
Hi - apologies, have just pushed up some changes.

Separated all tests into new files in the same directory as the original tests.